pkg/virtctl/pause/pause.go

```diff
@@ -122,7 +124,18 @@ func (vc *VirtCommand) Run(cmd *cobra.Command, args []string) error {
 			}
 			err = virtClient.VirtualMachineInstance(namespace).Pause(vmiName)
 			if err != nil {
-				return fmt.Errorf("Error pausing VirtualMachineInstance %s: %v", vmiName, err)
+				if errors.IsNotFound(err) {
+					runningStrategy, invalid := vm.RunStrategy()
+					if invalid != nil {
+						return invalid
+					}
+					if runningStrategy == kubevirtV1.RunStrategyHalted {
+						return fmt.Errorf("Error pausing VirtualMachineInstance %s. VirtualMachine %s is not set to run", vmiName, vmiName)
```

Comment:
nit: no need to repeat the VM name. Maybe just `Error pausing VirtualMachineInstance %s, it is not running`?

Reply:
Just for consistency ... as we always return it with name and to not confuse users.

Reply:
sure, but you do it twice here ;)

Reply:
Because of VMI and VM, not sure if they can be different ...

Reply:
ah, I see, fair point. However, you need to use `vm.Name` instead of `vmiName` for the 2nd parameter then

Reply:
https://github.com/kubevirt/kubevirt/blob/f77ae94e563d36424c65352826b848a31c58c9fa/pkg/virtctl/pause/pause.go#L121
Seems like they are same.... should I change it ?

Reply:
vmiName is potentially updated right after that line

Reply:
so this is the last one, everything else looks to me on the latest commit :+1:
Travis failed downloading a dependency, I restarted the job, I guess it will fine.